generate-pretty-manifest.js

```diff
@@ -0,0 +1,14 @@
+const { getMostRecentManifest } = require('./helpers.js');
+const fs = require('fs');
+const mostRecentManifestLoaded = require(`./${getMostRecentManifest()}`);
+
+let inventoryItem = mostRecentManifestLoaded.DestinyInventoryItemDefinition;
+let sources = mostRecentManifestLoaded.DestinyCollectibleDefinition;
+
+console.log("writing pretty printed manifest to ./output/prettymanifest.json");
+
+fs.writeFile("./output/prettymanifest.json",  JSON.stringify(mostRecentManifestLoaded, null, "  "), 'utf8', function(err) {
```

Comment:
writeFile(mostRecentManifestLoaded, './output/prettymanifest.json');

Reply:
JSON.stringify happens automatically unless you specify false as the 3 parameter

Reply:
After #9 
`writeFilePretty('./output/prettymanifest.json', mostRecentManifestLoaded);`